xml/favicons.xml

```diff
@@ -411,10 +420,10 @@
   </fingerprint>
 
   <fingerprint pattern="^1c4201c7da53d6c7e48251d3a9680449$">
-    <description>Nagios</description>
+    <description>Nagios (3.0.2)</description>
```

Comment:
Just wanted to confirm if this change is correct? It looks like this favicon hasn't changed in 17 years:

```
$ curl -s -L https://raw.githubusercontent.com/NagiosEnterprises/nagioscore/master/html/images/favicon.ico | md5sum
1c4201c7da53d6c7e48251d3a9680449  -
```

So I'm not sure if adding the version here is cromulent 👀

Reply:
Ah yes, that makes sense I added that there after trying to merge some duplicate entries. Thank you!